src/PowerShellEditorServices/Services/Analysis/AnalysisService.cs

```diff
@@ -959,7 +959,7 @@ private static DiagnosticSeverity MapDiagnosticSeverity(ScriptFileMarkerLevel ma
     /// <summary>
     /// Class to catch known failure modes for starting the AnalysisService.
     /// </summary>
-    public class AnalysisServiceLoadException : Exception
+    internal class AnalysisServiceLoadException : Exception
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this exception 'public'.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4979332)